### ISSUE TITLE
Hi, from a similar project

### ISSUE THREAD

**Issue by dai-shi** [excalideck/excalideck#7]
A competitor, or a friend? Anyway, here's some links.
- https://github.com/dai-shi/excalidraw-claymate
- https://github.com/scastiel/excalidraw-slides

**Comment by pscanf** [comment]
Hello 👋 🙂 , and thanks for the links!

- [scastiel/excalidraw-slides](https://github.com/scastiel/excalidraw-slides): I had actually seen this project before starting Excalideck, but I saw it more as a proof of concept, and having a slightly different workflow in mind, I decided to just start from zero

- [dai-shi/excalidraw-claymate](https://github.com/dai-shi/excalidraw-claymate): looks very interesting! Especially the animation feature, in the future that would be nice to have in Excalideck as well

> A competitor, or a friend? 

Not mutually exclusive 🙂 , but in general I'm very open to join forces with other projects.
